projects/packages/forms/src/blocks/contact-form/components/jetpack-field-controls.js

```diff
@@ -163,105 +157,260 @@ const JetpackFieldControls = ( {
 					onClick={ () => setAttributes( { required: ! required } ) }
 				/>
 			</BlockControls>
-
 			<InspectorControls>
 				<PanelBody title={ __( 'Manage Responses', 'jetpack-forms' ) }>
 					<JetpackManageResponsesSettings isChildBlock />
 				</PanelBody>
 				<PanelBody title={ __( 'Field Settings', 'jetpack-forms' ) }>
 					<>{ fieldSettings }</>
 				</PanelBody>
-				<PanelColorSettings
-					title={ __( 'Color', 'jetpack-forms' ) }
-					initialOpen={ false }
-					colorSettings={ colorSettings }
-				/>
-				<PanelBody title={ stylesPanelTitle } initialOpen={ false }>
-					<BaseControl>
+			</InspectorControls>
+			<JetpackFieldDimensionControls
```

Comment:
Similar to the checkbox/consent field suggestions, if we want to mimic the core block support panels perhaps we need to also mimic that order, which would have Dimensions after Color and Typography?